### PR TITLE
chore: add cyclone web user guide

### PR DESCRIPTION
**Pull request by cd1989**
**What this PR does / why we need it**:

Add Cyclone Web user guide.

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @zhujian7 @supereagle @qiuxiaolian @huxiangtao 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: docs/user-guide-web.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# Cyclone Web User Guide
```
Comment (supereagle) [inline docs/user-guide-web.md]: Use `doctoc` to generate an outline.

File: docs/user-guide-web.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+- Projects: Workflows are managed in projects, it's the main component of Cyclone web
+- Templates: Stages templates for Cyclone
+- Integrations: Integrated external services like Github, Docker registry.
+- Resource Types: Manage resource types Cyclone supported
+
+### Overview
+
+![overview](./images/screenshots/overview.png)
+
+### Resource Types
```
Comment (supereagle) [inline docs/user-guide-web.md]: I'd prefer that the order of component details keep consistent with that in Introduction.

File: docs/user-guide-web.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+Resource in Cyclone is an important concept, it describes external data to be used as input/output to Cyclone workflows.
```
Comment (supereagle) [inline docs/user-guide-web.md]: There is no need to emphasize the importance of `Resource` concept, as other concepts are also important.

```suggestion
Resource describes external data to be used as input/output of Cyclone workflows.
```

File: docs/user-guide-web.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+## Introduction
+
+Cyclone Web consists of five components:
+
+- Overview: Overview of the Cyclone platform
+- Projects: Workflows are managed in projects, it's the main component of Cyclone web
+- Templates: Stages templates for Cyclone
+- Integrations: Integrated external services like Github, Docker registry.
+- Resource Types: Manage resource types Cyclone supported
```
Comment (supereagle) [inline docs/user-guide-web.md]:
```suggestion
- Resource Types: Manage resource types Cyclone supports
```

File: docs/user-guide-web.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+### Projects
+
+Workflows in Cyclone are managed by projects. Project is a logical group of workflows. Users can define some common properties in project to be used by workflows belonging to it.
+
+![project detail](./images/screenshots/project-detail.png)
+
+3 kinds of resources are managed in project:
+
+- **Stages**  Building block of workflow. It defines how a stage works.
```
Comment (supereagle) [inline docs/user-guide-web.md]: * Missing `:`
* `Building block` -> `Execution unit`

```suggestion
- **Stages**: Execution unit of workflow. It defines how a stage works.
```

**Comment by zhujian7** [comment]
LGTM

File: docs/user-guide-web.md (hunk 2 above)
Comment (cd1989) [reply docs/user-guide-web.md]: ok

**Pull request merged**